wagtail/tests/utils/form_data.py

```diff
@@ -42,35 +42,6 @@ def nested_form_data(data):
     return {'-'.join(key): value for key, value in _nested_form_data(data)}
 
 
-def streamfield(items):
```

Comment:
This is a public documented function, and if we're removing it, we need to remove it from the docs and add a release note:

https://docs.wagtail.io/en/v2.4/advanced_topics/testing.html#wagtail.tests.utils.form_data.streamfield

However, probably a better idea is to keep it here, and rewrite it to output the JSON value instead - that way, we avoid making a breaking change, and no longer need to update our own tests (and the tuple syntax is slightly more human-friendly than having devs write JSON directly anyhow).

Reply:
OK, I revert that.

Reply:
Introduced back as https://github.com/wagtail/wagtail/pull/4942/commits/821d7e924e5fd9c7a90e137be9f9054f6d9bfac1. Do you think that’s enough?